integrations/access/accessrequest/app.go

```diff
@@ -144,9 +153,13 @@ func (a *App) run(ctx context.Context) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-
-	if err := a.initAccessMonitoringRulesCache(ctx); err != nil {
-		return trace.Wrap(err)
+	if len(acceptedWatchKinds) == 0 {
+		return errors.New("no acceptedWatchKinds returned")
+	}
+	if slices.Contains(acceptedWatchKinds, types.KindAccessMonitoringRule) {
```

Comment:
Nit: Add a comment here explaining why monitoring rule may not be among watch kinds.